Solutions/archTIS/Data/Solution_archTIS.json

```diff
@@ -1,15 +1,17 @@
 {
-  "Name": "archTIS",
-  "Author": "Nikhil Tripathi - v-ntripathi@microsoft.com",
+  "Name": "NC Protect Data Connector",
+  "Author": "archTIS",
   "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/archTIS/Data%20Connectors/Logo/NCProtectIcon.svg\"width=\"75px\"height=\"75px\">",
-  "Description": "[NC Protect Data Connector (archtis.com)](https://info.archtis.com/get-started-with-nc-protect-sentinel-data-connector) provides the capability to ingest user activity logs and events into Azure Sentinel. The connector provides visibility into NC Protect user activity logs and events in Azure Sentinel to improve monitoring and investigation capabilities.",
```

Comment:
Hi @v-marimanda, please change description text from `Azure Sentinel` to `Microsoft Sentinel`, also change text for Data Connector also